code/datums/status_effects/debuffs.dm

```diff
@@ -9,11 +9,12 @@
 	alert_type = null
 
 /datum/status_effect/incapacitating/on_creation(mob/living/new_owner, set_duration)
+	if(new_owner.status_flags & GODMODE)
+		return qdel(src)
```

Comment:
Why are you returning qdel in on creation in 2 places

Reply:
What do you mean? intoxication is a stacking effect not a incapacitating. Or do you mean I should have it on the parent proc and have some if(var) check?

Reply:
qdel always returns null